local-rendering.md

```diff
@@ -1 +1,36 @@
-# Serving the site locally
\ No newline at end of file
+# Serving the site locally
+
+## 1. Set the path of your content folder
+
+The first step to rendering your content folder with `NEBULA` is to set the `CONTENT_PATH` environment variable to the path of your content repository, *i.e.*,
+
+```bash
+export CONTENT_PATH="path-to-your-content-repository"
+```
+
+## 2. Setup `npm`
```

Comment:
As mentioned above, I think this is better as a second step. Also we cannot assume they have npm installed, so before this step, the user should check whether npm is installed and otherwise get a link to the documentation on installing npm.

Reply:
Done!